btf/types.go

```diff
@@ -67,13 +71,73 @@ func (ts types) ByID(id TypeID) (Type, error) {
 	return ts[id], nil
 }
 
+type Skipper func(Type) bool
```

Comment:
Nice, could use this for postorderTraversal as well. Should have documentation if we decide to export it though, otherwise there is no point?

Nit: should this be `SkipFn` instead to distinguish from an interface?